### ISSUE TITLE
Sentry: null is not an object (evaluating 'e.scrollTop')

### ISSUE THREAD

**Issue by riz007** [mdbootstrap/perfect-scrollbar#980]
After upgrading vue-perfect-scrollbar to 0.2.1, I am receiving this error a lot (116k times) in Sentry:
Browser: Safari
OS: Mac OS X / iOS
User-Agent
`Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_6) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/14.0 Safari/605.1.15
`

![image](https://user-images.githubusercontent.com/53720206/99220488-5d4fd480-2811-11eb-85c5-24d5057f6e7e.png)

```
 * perfect-scrollbar v1.4.0
 * (c) 2018 Hyunje Jun
 * @license MIT
{snip} t.containerWidth=e.clientWidth,t.containerHeight=e.clientHeight,t.contentWidth=e.scrollWidth,t.contentHeight=e.scrollHeight,e.contains(t.scr {snip}

```
Any ideas why this could be happening would be greatly appreciated. Is there any workaround for this?

**Comment by XEngine** [comment]
Have you solved this issue ?

**Comment by hmoffatt** [comment]
I have tons of these also from my users (via bugsnag), all from Safari and iOS web views. Also using vue2-perfect-scrollbar.

**Comment by riz007** [comment]
> Have you solved this issue ?

Yes. I simply forked the repo and applied the fix there. Someone already created a PR here: https://github.com/mdbootstrap/perfect-scrollbar/pull/977
They are yet to merge it so I found forking and fixing is the only way to stop this error which was the root cause of my Sentry quota exceed.

**Comment by riz007** [comment]
> I have tons of these also from my users (via bugsnag), all from Safari and iOS web views. I have perfect-scrollbar 1.5.0 and am also using vue2-perfect-scrollbar 1.5.0.

You can fork the repos and apply the fixes there. Someone already created a PR here: https://github.com/mdbootstrap/perfect-scrollbar/pull/977 but I am not sure when they will merge it.

**Comment by hmoffatt** [comment]
Thanks @riz007 will deploy that and hope the error reports go away.

**Comment by hmoffatt** [comment]
Seems to have done the trick.

**Comment by nh2** [comment]
A new release was made that's supposed to include the fix but it doesn't seem to work for me because of https://github.com/mdbootstrap/perfect-scrollbar/pull/977#issuecomment-882065525
